### PR TITLE
New activation functions

### PR DESCRIPTION
**Pull request by exowanderer**
```
Not to be presumptuous, but `master` is the only branch that I could think to point the PR.
```
I've added the activation functions that I use the most (SeLU, eLU, LeLU) -- when manually setting the hyperparameters. These come from various other papers. I nominally run a DNN with each of the activation functions, and the select the one that has the best validation accuracy in the least number of iterations.  This would likely be better optimized when using comparative activation functions with NEAT.

Edits:

Added three new to `neat/activation.py`: 

```python
def relu_activation(z):
    return z if z > 0.0 else 0.0

def elu_activation(z):
    return z if z > 0.0 else math.exp(z) - 1

def selu_activation(z, lam=1.0507009873554804934193349852946, alpha=1.6732632423543772848170429916717):
    return lam*z if z > 0.0 else lam*alpha*(math.exp(z) - 1)
```

The funky values for `lam` and `alpha` values come from the paper: 

https://arxiv.org/pdf/1706.02515.pdf

but I found them on Reddit at 

https://www.reddit.com/r/MachineLearning/comments/6g5tg1/r_selfnormalizing_neural_networks_improved_elu/

### CODE REVIEW

**Comment by evolvingfridge** [comment]
If you interested in activation and aggregation functions take a look @drallensmith fork [here](https://github.com/drallensmith/neat-python/blob/multiparam_funcs/neat/activations.py).

**Comment by CodeReclaimers** [comment]
Apologies for the long silence--I've been overwhelmed with other stuff and just kind of let this fall idle.  I appreciate the patch and will take a look as soon as I get some free time. Thanks!

**Comment by CodeReclaimers** [comment]
Thanks for the patch!
